samples/highcharts/accessibility/internationalization/demo.js

```diff
@@ -0,0 +1,287 @@
+            exportButtonTitle: 'Eksporter',
+            printButtonTitle: 'Print',
+            printChart: 'Skriv ut diagram',
+            viewFullscreen: 'Se i fullskjerm',
+            rangeSelectorFrom: 'Fra',
+            rangeSelectorTo: 'Til',
+            rangeSelectorZoom: 'Zoom',
+            downloadPNG: 'Last ned som PNG-bilde',
+            downloadJPEG: 'Last ned som JPEG-bilde',
+            downloadPDF: 'Last ned som PDF-dokument',
+            downloadSVG: 'Last ned som SVG-bilde',
+            title: 'AAPL Aksjekurs'
+        }
+    },
+    de: {
+        lang: {
+            accessibility: {
+                rangeSelector: {
+                    maxInputLabel: 'Endpunkt auswählen',
+                    minInputLabel: 'Startpunkt auswählen'
+                },
+                chartContainerLabel: 'Interaktives Highcharts-Diagramm',
+                exporting: {
+                    chartMenuLabel: 'Diagramm-Menü',
+                    exportRegionLabel: 'Diagramm-Menü',
+                    menuButtonLabel: 'Diagramm-Menü ansehen'
+                },
+                svgContainerLabel: 'Interaktives Diagramm',
+                screenReaderSection: {
+                    beforeRegionLabel: 'Screenreader-Information des Diagramms',
+                    endOfChartMarker: 'Ende des interaktiven Diagramms'
+                }
+            },
+            decimalPoint: ',',
+            thousandsSep: '.',
+            loading: 'Daten werden geladen...',
+            months: ['Januar', 'Februar', 'März', 'April', 'Mai', 'Juni', 'Juli', 'August', 'September', 'Oktober', 'November', 'Dezember'],
+            weekdays: ['Sonntag', 'Montag', 'Dienstag', 'Mittwoch', 'Donnerstag', 'Freitag', 'Samstag'],
+            shortMonths: ['Jan', 'Feb', 'Mrz', 'Apr', 'Mai', 'Jun', 'Jul', 'Aug', 'Sep', 'Okt', 'Nov', 'Dez'],
+            exportButtonTitle: "Exportieren",
+            printButtonTitle: "Drucken",
+            printChart: 'Diagramm ausdrucken',
+            viewFullscreen: 'Im Vollbild anschauen',
+            rangeSelectorFrom: "Von",
+            rangeSelectorTo: "Bis",
+            rangeSelectorZoom: "Zeitraum",
+            downloadPNG: 'Download als PNG-Bild',
+            downloadJPEG: 'Download als JPEG-Bild',
+            downloadPDF: 'Download als PDF-Dokument',
+            downloadSVG: 'Download als SVG-Bild',
+            resetZoom: "Zoom zurücksetzen",
+            resetZoomTitle: "Zoom zurücksetzen",
+            title: 'AAPL Aktienkurs'
+        }
+    }
+};
+
+Highcharts.getJSON('https://demo-live-data.highcharts.com/aapl-c.json', data => {
+
+    const chartOptions = {
+        chart: {
+            style: {
+                fontFamily: 'Arial'
+            }
+        },
+        accessibilty: {
+            screenReaderSection: {
+                beforeChartFormat: 'Financial line chart showing AAPL stock prices, with an additional navigator series showing an overview of the stock.'
+            }
```

Comment:
I think we should add these options as well for a slightly better screen reader experience:
https://api.highcharts.com/highcharts/accessibility.series.pointDescriptionEnabledThreshold - set this to 30
https://api.highcharts.com/highcharts/accessibility.landmarkVerbosity - set this to `'one'`